Automatic UI Generator/gadget.xml

```diff
@@ -0,0 +1,469 @@
+                    '}\n'+
+                    '.btn {\n'+
+                    'background-color: white;\n'+
+                    'color: black;\n'+
+                    '}\n'+
+                    '.button-on {\n'+
+                    'background-color: #6262FF;\n'+
+                    '}\n'+
+                    'h4 { \n'+
+                    'margin-top: 5px;\n'+
+                    'margin-bottom: 0px;\n'+
+                    'font-size: 16px;\n'+
+                    '}\n'+
+                    '</style>\n'+
+                    '</head>\n';
+
+                var onMessage = 'ws.onmessage = function(event) {\n'+
+                    'var data = event.data;\n'+
+                    'var msg;\n'+
+                    'try {\n'+
+                    'msg = JSON.parse(data);\n'+
+                    'if (msg.method == "getSensorData") {\n'+
+                    'if (msg.responseData) {\n'+
+                    'if (booleanArray.indexOf(msg.sensorId) > (-1)) {\n'+
+                    'if (msg.responseData.data[0]){\n'+
+                    '$("#"+msg.sensorId).css("background-color", "#228B22");\n'+
+                    '} else {\n'+
+                    '$("#"+msg.sensorId).css("background-color", "white");\n'+
+                    '}\n'+
+                    '} else {\n'+
+                    '$("#"+msg.sensorId).html(msg.responseData.data[0]);\n'+
+                    '}\n'+
+                    '}\n'+
+                    '}\n'+
+                    '} catch(e) {\n'+
+                    'console.log(e);\n'+
+                    '}\n'+
+                    '};\n';
+
+                var showHideElements = 'var showHideElements = function (chosenOption) {\n'+
+                    '$.each(allSensorsActuatorsArray, function(index, element) {\n'+
+                    'if (configurationArray[chosenOption].indexOf(element)>(-1)) {\n'+
+                    'if(slidersArray.indexOf(element)>(-1)){\n'+
+                    '$("#"+element).parent().show();\n'+
+                    '} else {\n'+
+                    '$("#"+element).show();\n'+
+                    '$("#span"+element).show();\n'+
+                    '}\n'+
+                    '}\n'+
+                    'else {\n'+
+                    'if(slidersArray.indexOf(element)>(-1)){\n'+
+                    '$("#"+element).parent().hide();\n'+
+                    '} else {\n'+
+                    '$("#"+element).hide();\n'+
+                    '$("#span"+element).hide();\n'+
+                    '}\n'+
+                    '}\n'+
+                    '});\n'+
+                    '}\n';
+
+                var tail = '});\n'+
+                    '</scri'+'pt>\n'+
+                    '</body>\n'+
+                    '</html>\n'+
+                    ']'+']>\n'+
+                    '</'+'Content>\n'+
+                    '</'+'Module>\n';
+
+      });
+    </script>
+</body>
+</html>
+]]>
+</Content>
+</Module>
```

Comment:
@hagopj13 
![screen shot 2016-09-27 at 11 41 15](https://cloud.githubusercontent.com/assets/2853659/18868265/d41f229e-84a7-11e6-9489-ac1e5b10fbcb.png)
Is the white part a section of the styling or there something not working properly?

i guess the same comments go for the html version to keep them consistent.
[1] Please rename the branch to have only small letters.
[2] Rename the folder containing the gadget and html files to have small letters and no spaces.
[3] And to avoid having both pull requests in one, when renaming the branch make sure you pull master and based on it you create the new branch for this pull request.